File: src/sdk-reference/cpp/1/document/mDelete/snippets/m-delete.cpp
std::vector<std::string> ids;

ids.push_back("some-id");
ids.push_back("some-other-id");

try {
  kuzzle->document->create("nyc-open-data", "yellow-taxi", "some-id", "{}");
  kuzzle->document->create("nyc-open-data", "yellow-taxi", "some-other-id", "{}");

  std::vector<std::string> deleted = kuzzle->document->mDelete("nyc-open-data", "yellow-taxi", ids);

  std::cout << "Successfully deleted " << deleted.size() << " documents" << std::endl;
} catch (kuzzleio::KuzzleException e) {
  std::cerr << e.getMessage() << std::endl;
}

File: src/sdk-reference/csharp/1/document/mDelete/snippets/m-delete.cs
List<string> ids = new List<string>();

ids.Add("some-id");
ids.Add("some-other-id");

try {
  kuzzle.document.create("nyc-open-data", "yellow-taxi", "some-id", "{}");
  kuzzle.document.create("nyc-open-data", "yellow-taxi", "some-other-id", "{}");

  List<string> deleted = kuzzle.document.mDelete("nyc-open-data", "yellow-taxi", ids);

  Console.WriteLine("Successfully deleted " + deleted.Count + " documents");
} catch  {
  Console.Error.WriteLine("");
}